tests/common/common.go

```diff
@@ -0,0 +1,94 @@
+package common
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"mime/multipart"
+	"net/http"
+	"net/url"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestDataPath test data path
+const TestDataPath string = "../presidio-tester/cmd/presidio-tester/testdata/"
+const host = "localhost:8080"
+
+//GeneratePayload generate payload
+func GeneratePayload(name string) []byte {
+	payload, err := ioutil.ReadFile(TestDataPath + name)
+	if err != nil {
+		panic(err)
+	}
+	return payload
+}
+
+//InvokeHTTPRequest invoke http request
+func InvokeHTTPRequest(t *testing.T, path string, method string, payload []byte) string {
```

Comment:
duplicate code that exists on tester.go

Reply:
The signature is different. tester.go is not using the `t *testing.T` required for unit test